### PR TITLE
BlockChain as Service new Azure4Students template

### PR DESCRIPTION
**Pull request by Krisnatagoras**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 201-blockchain-asaservice/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            }  
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Blockchain/blockchainMembers",
+            "apiVersion": "2018-06-01-preview",
+            "name": "[parameters('bcMemberName')]",
+            "location": "[if(or(equals(parameters('location'),'southcentralus'),equals(parameters('location'),'southeastasia'),equals(parameters('location'),'westeurope'),equals(parameters('location'),'northeurope'),equals(parameters('location'),'westus2')),parameters('location'),'eastus')]",
```
Comment (bmoore-msft) [inline 201-blockchain-asaservice/azuredeploy.json]: what problem are we trying to solve here?

File: 201-blockchain-asaservice/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "bcMemberName": {
+            "value": "GEN-UNIQUE"
+        },
+        "memberPassword": {
+           "value": "SecureP4$$w0rd"
```
Comment (bmoore-msft) [inline 201-blockchain-asaservice/azuredeploy.parameters.json]: use GEN-PASSWORD

File: 201-blockchain-asaservice/azuredeploy.json (hunk 1 above)
Comment (Krisnatagoras) [reply 201-blockchain-asaservice/azuredeploy.json]: The idea was treat the available locations. But I will use the same aproach that we used with Machine Learning.

**Comment by leestott** [comment]
@bmoore-msft  so this Error: Code=LocationNotAvailableForResourceType; Message=The provided location 'westus' is not available for resource type 'Microsoft.Blockchain/blockchainMembers'. List of available regions for the resource type is 'eastus,southeastasia,westeurope,northeurope,westus2,japaneast'. What is the most elegant way of specific region?

**Pull request merged**